packages/kmod-6.12-nvidia-r570/nvidia-tesla-tmpfiles.conf

```
@@ -0,0 +1,5 @@
+ /etc/drivers/nvidia-tesla.toml
```

This needs a C at the beginning
```suggestion
C /etc/drivers/nvidia-tesla.toml
```